js/plugins/google-cloud/src/utils.ts

```diff
@@ -26,6 +26,15 @@ export function extractOuterFlowNameFromPath(path: string) {
   return flowName ? flowName[1] : '<unknown>';
 }
 
+export function extractOuterFeatureNameFromPath(path: string) {
+  if (!path || path === '<unknown>') {
+    return '<unknown>';
+  }
+
+  const flowName = path.match('/{(.+),t:(flow|action|prompt)}+');
```

Comment:
Are these the only 3 top-level types? Is "helper" a thing now?

Reply:
Yes, it would be good to include helper.